test/queries/queries-good.sql

```diff
@@ -70,11 +70,13 @@ COPY students FROM 'file_path' WITH FORMAT TBL;
 COPY students FROM 'file_path' WITH FORMAT CSV;
 COPY students FROM 'file_path' WITH FORMAT BIN;
 COPY students FROM 'file_path' WITH FORMAT BINARY;
+COPY good_students FROM 'file_path' WHERE grade > (SELECT AVG(grade) from alumni);
 COPY students TO 'student.tbl';
 COPY students TO 'file_path' WITH FORMAT TBL;
 COPY students TO 'file_path' WITH FORMAT CSV;
 COPY students TO 'file_path' WITH FORMAT BIN;
 COPY students TO 'file_path' WITH FORMAT BINARY;
+COPY (SELECT firstname, COUNT(*) FROM students GROUP BY firstname) TO 'student_names.csv';
```

Comment:
Why not `file_path`?

Reply:
Besides that nothing to add.

I am nonetheless wondering, if we have tested all possible (or at least relevant) cases. But tbh, I have used queries on exports and I wasn't even aware it existed.

I guess an empty results is just a CSV file (if requested) with a header, right?

Reply:
Just something figurative here for the example query. Can change if requested.

I just hooked the existing select statement into the export statement: ﻿https://github.com/hyrise/sql-parser/blob/396fd59aa6bd154e330eb485fee1473245a6df84/src/parser/bison_parser.y#L475
The select statement should be tested accordingly in [`select_tests.cpp`](https://github.com/hyrise/sql-parser/blob/396fd59aa6bd154e330eb485fee1473245a6df84/test/select_tests.cpp). Same goes for the `WHERE`clause of the import statement.

Last question: The Hyrise exporter will write the header regardless of the content (so an empty table results in the written header - however, we do not write CSV headers but create an extra file containing the schema information). If the desired table does not exist, we fail.